src/aead/aes.rs

```diff
@@ -291,8 +291,8 @@ impl Key {
         }
     }
 
-    pub fn new_mask(&self, sample: Block) -> [u8; 5] {
-        let block = self.encrypt_block(sample);
+    pub fn new_mask(&self, sample: [u8; BLOCK_LEN]) -> [u8; 5] {
```

Comment:
I think we move put `SAMPLE_LEN` in `aead::quic`. I thinki we should also define `pub type Sample = [u8; SAMPLE_LEN]` there. Then all these uses of `Block` would be replaced by `Sample`.